core/templates/dev/head/pages/exploration-editor-page/improvements-tab/answer-details-improvement-card/answer-details-improvement-card.directive.html

```diff
@@ -1,9 +1,13 @@
-<div>
+<div class="row oppia-state-name-box">
   <b>State Name:</b>
   <span class="protractor-test-answer-details-state"><[getStateName()]></span>
 </div>
-<br>
-<div>
+<div class="row">
   <b>Number of answer details received:</b>
   <span class="protractor-test-answer-info-count"><[getLearnerAnswerInfoCount()]></span>
 </div>
+<style>
+  .oppia-state-name-box {
```

Comment:
Prefix with `answer-details-improvement-card` (to scope it to this directive)?

Reply:
Done